demo/filter-dropdown/filter-dropdown.html

```diff
@@ -22,77 +22,73 @@
 			<h2>d2l-labs-filter-dropdown with dom-repeat data and event hookups</h2>
 
 			<d2l-demo-snippet>
-				<template>
```

Comment:
I don't think the `template`s here needed to be removed because these are demo files for Polymer components lol.  But I don't think they're doing anything useful.